<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScopePaneForm</class>
 <widget class="QWidget" name="ScopePaneForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="scopeArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="averageCheckBox">
        <property name="toolTip">
         <string>Enable/disable data averaging</string>
        </property>
        <property name="text">
         <string>Averaging</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>240</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="horizComboBox">
        <property name="toolTip">
         <string>Select a horizontal scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="vertComboBox">
        <property name="toolTip">
         <string>Select a vertical scale (amplification)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
